experiments/scripts/delete_old_parquet.py

```diff
@@ -0,0 +1,58 @@
+from google.cloud import storage
+from pprint import pprint
+import sys
+from pathlib import Path
+sys.path.insert(0, f'{Path.cwd()}/')
+from config import COLS_SELECT, COLS_INTEREST, BUCKET_NAME
+
+
+
+def list_parquet_files(bucket_name=BUCKET_NAME):
+    """
+    Gathers the data of parquet files already stored in GCS
+    """
+    # init GCS client, get all blobs in bucket
+    client = storage.Client()
+    bucket = client.get_bucket(bucket_name)
+    blobs = bucket.list_blobs(prefix='dailies/')
+    
+    # list of dates with "cleaned up" raw data
+    parquet_files = [
+        str(i).split(',')[1].strip() for i in blobs
+        if str(i).split(',')[1].endswith('_tweets.parquet')
+    ]
+    
+    return parquet_files
+
+
+def delete_blob(bucket_name, blob_name):
+    """Deletes a blob from the bucket."""
+    # bucket_name = "your-bucket-name"
+    # blob_name = "your-object-name"
+
+    storage_client = storage.Client()
+
+    bucket = storage_client.bucket(bucket_name)
+    blob = bucket.blob(blob_name)
+    # print(blob.exists())
+    blob.delete()
+
+    print(f'Blob {blob_name} deleted.\n')
+
+
+def main():
+    """
+    TBD
+    """
+    bucket_name = BUCKET_NAME
```

Comment:
It already has a name. Why give it another?

Reply:
`delete_old_parquet` was a one-off script that I generated to go through each daily sub-folder and remove an old file that was no longer relevant (versus having to go through it "by hand" via the console). But I will take note of this for future reference.